modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -2110,22 +2136,34 @@ boolean addTrailingCells(boolean fillEmptyCells) {
         return filledWithNonEmpty;
     }
 
-    void reconfigureCells() {
+    /**
+     * Inform the VirtualFlow that a layout should be done and cell contents are the same.
+     */
+    protected void reconfigureCells() {
         needsReconfigureCells = true;
         requestLayout();
     }
 
-    void recreateCells() {
+    /**
+     * Inform the VirtualFlow that a layout should be done and cell factory has changed.
+     */
+    protected void recreateCells() {
         needsRecreateCells = true;
         requestLayout();
     }
 
-    void rebuildCells() {
+    /**
+     * Inform the VirtualFlow that a layout should be done and cell contents have changed.
+     */
+    protected void rebuildCells() {
         needsRebuildCells = true;
         requestLayout();
     }
 
-    void requestCellLayout() {
+    /**
+     * Inform the VirtualFlow that a layout should be done and only the cell layout will be triggered.
+     */
+    protected void requestCellLayout() {
```

Comment:
All the above methods should be made protected so that the developper can inform the VirtualFlow of some change, and trigger a layout. 
This is part of the basic toolkit when custom rows are inserted and may have some impact that the original VirtualFlow is not aware of.